pages/common/npm-install.md

```diff
@@ -0,0 +1,20 @@
+# npm install
+
+> Install node packages.
+> More information: <https://docs.npmjs.com/cli/commands/npm-install>.
+
+- Install dependencies listed in package.json:
+
+`npm install`
+
+- Download a specific version of a package and add it to the list of dependencies in `package.json`:
+
+`npm install {{package_name}}@{{version}}`
+
```

Comment:
- Install a package and update package.json:

`npm install {{package_name}} --save`

Reply:
add this command

Reply:
That seems to be the default behavior and is not needed after npm 5.